UNRELEASED.md

```diff
@@ -58,6 +58,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 - Added support for React hooks in Storybook ([#1665](https://github.com/Shopify/polaris-react/pull/1665))
 - Created `toBeDisabled`, `mountWithContext` and added custom testing matchers ([#1596](https://github.com/Shopify/polaris-react/pull/1596))
 - Enabled strict mode in TypeScript ([#1883](https://github.com/Shopify/polaris-react/pull/1883))
+- Moved to `unpkg.com` for our CDN CSS assets, instead of using `sdks.shopifycdn.com`. Existing URLs will continue to work but new versions will only be available on `unpkg.com` ([#1960](https://github.com/Shopify/polaris-react/pull/1960))
```

Comment:
Is there a section in the change log for breaking changes in v4? Will this change also be included in the upgrade instructions for v4?

Reply:
There is a breaking changes section. Do you think it belongs there more? I was torn as existing links will still continue to work, it's only if you want the new stuff the URL has changed.

Reply:
I'll also be mentioning this in the migration guide doc that is being written in https://github.com/Shopify/polaris-react/pull/1945

Reply:
Agreed--since the old links continue to work and it's just moving forward from v4 on that the domain changes, it wouldn't make sense to call this a breaking change 👍